plugins/woocommerce-admin/client/activity-panel/panels/help.js

```diff
@@ -235,10 +237,15 @@ function getTaxItems( props ) {
 	}
 
 	const { additionalData } = task;
-	const { woocommerceTaxCountries = [], taxJarActivated } = additionalData;
+	const {
+		woocommerceTaxCountries = [],
+		taxJarActivated,
+		woocommerceTaxActivated,
+	} = additionalData;
 
 	const showWCS =
 		! taxJarActivated && // WCS integration doesn't work with the official TaxJar plugin.
+		! woocommerceTaxActivated &&
```

Comment:
Prevents WCS&T-related setup instructions from being displayed in the help sidebar when viewing tax-related onboarding tasks with the WC Tax plugin active:

<img width="744" alt="Screenshot 2024-06-06 at 13 56 19" src="https://github.com/woocommerce/woocommerce/assets/1759681/4efb3b43-f154-4c9d-b480-72b987841d97">

Note that despite the help item saying "WooCommerce Tax", it actually refers to WCS&T's tax functionality.